.github/workflows/bench_applications.yml

```diff
@@ -24,6 +24,7 @@ concurrency:
 
 jobs:
   bench:
+    if: false # Currently broken. See https://github.com/risc0/risc0/issues/1013
```

Comment:
I think it's OK for this to fail

Reply:
Removed this change in 6b167135198f97d7b4ad829d00c699e6f026ab16